app/bundles/ApiBundle/ApiEvents.php

```diff
@@ -60,4 +60,22 @@ final class ApiEvents
      * @var string
      */
     public const API_ON_ENTITY_POST_SAVE = 'mautic.api_on_entity_post_save';
+
+    /**
+     * The mautic.api_pre_serialization_context event is thrown before the serialization context is created for the view.
```

Comment:
the word `thrown` sound weird here (i know the other events in this and other files have similar comments)
```suggestion
     * The mautic.api_pre_serialization_context event is dispatched before the serialization context is created for the view.
```